_1327/shortlinks/views.py

```diff
@@ -0,0 +1,70 @@
+from django.contrib import messages
+from django.core.exceptions import PermissionDenied
+from django.http import HttpResponse
+from django.shortcuts import get_object_or_404, Http404, redirect, render
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+from .forms import ShortlinkForm
+from .models import Shortlink
+
+
+def shortlinks_index(request):
+	if not request.user.is_superuser:
```

Comment:
only superusers are allowed to create/edit shortlinks?
Is this really the intention?